docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Cluster Kyma installation
+type: Getting Started
+---
+
+## Overview
+
+This Getting Started guide shows developers how to to quickly deploy Kyma on a cluster. Kyma installs on a cluster using a proprietary installer based on a Kubernetes operator. The document provides prerequisites, instructions on how to install Kyma on a cluster and verify the deployment, as well as the troubleshooting tips.
+
+## Prerequisites
+
+**NOTE:** The public IPs and DNS records for Istio Ingress and the remote environments gateway must exist prior to the Kyma installation. Also, the Kyma installer does not support clusters on AWS because AWS does not support static IP assignment during ELB creation.
+
+Before you start the installation, check that you have access to a running cluster with the minimum Kubernetes version 1.10.
+
+To install Kyma, you need the following data:
```

Comment:
I think the punctuation in this section is not consistent - some are sentences that end with a period, some are not. 
I'd put a period at the end of each of the points even though they do not contain verbs. 

![image](https://user-images.githubusercontent.com/29116492/43716699-62d0aaf0-9985-11e8-9fdd-7aa3087b9f3c.png)

Reply:
I am not sure about it...it is inconsistent because of additional sentences there but is it enough to add it everywhere?... perhaps it would be better to add the info on additional reading "For details," somewhere at the end, after the bullet points and rephrase ``The domain name. For example, `kyma.example.com`.`` into ``The domain name such as`kyma.example.com` ``

Reply:
I'd just make it consistent so either with a period or w/o one for every point.